test-utils/src/crypto.rs

```diff
@@ -26,3 +26,14 @@ pub fn init_merkle_leaves(values: &[u64]) -> Vec<Word> {
 pub fn init_merkle_leaf(value: u64) -> Word {
     [Felt::new(value), Felt::ZERO, Felt::ZERO, Felt::ZERO]
 }
+
+/// Returns a remaining path key for a Sparse Merkle tree
+#[cfg(not(target_family = "wasm"))]
+pub fn get_remaining_key(mut key: Word, depth: u8) -> Word {
```

Comment:
minor nit: should we name this `get_smt_remaining_key()`?

Reply:
Makes sense; renamed